.github/workflows/ci.ym

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
You are specifying to use use Node version 14 here, but that's a quite old version by now. Node 14 was released in 2020 and stopped receiving updates in 2023. The current version of Node with long-term support (LTS) is v20. I'm guessing you've found an old tutorial and used the values from that, which is an easy mistake to make.